packages/wrangler/package.json

```diff
@@ -80,6 +81,7 @@
     "tmp-promise": "^3.0.3",
     "undici": "^4.11.1",
     "ws": "^8.3.0",
+    "xxhash-addon": "^1.4.0",
```

Comment:
if `xxhash-addon` is marked as an external in our build script, then this should be moved to `dependencies`.